OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -79,16 +79,16 @@ inline void ASSERT(bool cond,
  */
 void CHECK_STORAGE_AGAINST_STANDARD(OpenSim::Storage& result, 
                                     OpenSim::Storage& standard, 
-                                    OpenSim::Array<double> tolerances, 
+                                    std::vector<double> tolerances, 
                                     std::string testFile, 
                                     int testFileLine, 
                                     std::string errorMessage)
 {
```

Comment:
I think the parameters need to be received as `const T&`. I understand this PR is not introducing the problem but good to fix it if possible.